static/css/impala/site.less

```diff
@@ -10,6 +10,7 @@ html, body {
 
 #main-wrapper {
     background: #fff url(../../img/zamboni/global/bg-header.png) repeat-x;
+    color: #000;
```

Comment:
The background itself is specified here. If there is some page that doesn’t use `#main-wrapper` and doesn’t have background specified, it would create similar problem (colour set, but not a background).

Another way is moving it to `body` with additional white background, body has greater chance of having it’s background colour reset ([like here](https://github.com/mozilla/olympia/blob/d8aef0b88a5fd20d4413a37807e9ae79c7602fa8/static/css/zamboni/admin-mozilla.css#L41)) and the problem could return. Although it might be a better solution, I just am not sure which body rule to target.